lib/src/component/resize_sensor.dart

```diff
@@ -106,13 +107,32 @@ class ResizeSensorComponent extends UiComponent<ResizeSensorProps> with _SafeAni
   @override
   void componentWillUnmount() {
     super.componentWillUnmount();
-
+    if (ResizeObserver.supported) {
+      ResizeObserver.unobserve(findDomNode(this));
+    }
     cancelAnimationFrames();
   }
 
   @mustCallSuper
   @override
   void componentDidMount() {
+    if (ResizeObserver.supported) {
+      Element el = findDomNode(this);
+      _lastHeight = 0;
+      _lastWidth = 0;
+      ResizeObserver.observe(findDomNode(this), (el, x, y, w, h, t, b, l, r) {
```

Comment:
Use `el` instead of finding the node again.

I'd also suggest a class method vs an anonymous one.